main/src/ca/uwaterloo/flix/language/phase/jvm/BackendObjType.scala

```diff
@@ -164,6 +165,28 @@ object BackendObjType {
     ))
 
     def ArgField(index: Int): InstanceField = InstanceField(this.jvmName, IsPublic, NotFinal, s"arg$index", args(index))
+
+    def ToStringMethod: InstanceMethod = {
```

Comment:
How about we compute the string at compile-time instead? :)

Reply:
Oh yeah of course, I'm stuck in jvm

Reply:
Actually I was totally optimizing the amount of string constants of the class

Reply:
But one string would be shortest right? No instructions to save :)

Reply:
oh yeah theyre different classes. cant even find a made up excuse 😄